wurst/_handles/TimerDialog.wurst

```diff
@@ -0,0 +1,37 @@
+package TimerDialog
+import NoWurst
+import Colors
+
+public function timer.createTimerDialog() returns timerdialog
+    return CreateTimerDialog(this)
+
+public function timerdialog.destr() 
+    DestroyTimerDialog(this)
+
+public function timerdialog.setTitle(string title) 
+    TimerDialogSetTitle(this, title)
+
+public function timerdialog.setTitleColor(int red, int green, int blue, int alpha) 
+    TimerDialogSetTitleColor(this, red, green, blue, alpha)
+
+public function timerdialog.setTitleColor(colorA colorA) 
+    TimerDialogSetTitleColor(this, colorA.red, colorA.green, colorA.blue, colorA.alpha)
+
+public function timerdialog.setTimeColor(int red, int green, int blue, int alpha) 
+    TimerDialogSetTimeColor(this, red, green, blue, alpha)
+
+public function timerdialog.setTimeColor(colorA colorA) 
+    TimerDialogSetTimeColor(this, colorA.red, colorA.green, colorA.blue, colorA.alpha)
+
+public function timerdialog.setSpeed(real speedMultFactor) 
```

Comment:
Can you add a doc comment for what this is for? I'm not familiar with the native